app/uk/gov/hmrc/accessibilitystatementfrontend/TestOnlySourceConfigModule.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.accessibilitystatementfrontend
+
+import com.google.inject.AbstractModule
+import uk.gov.hmrc.accessibilitystatementfrontend.config.{SourceConfig, TestOnlySourceConfig}
+
+class TestOnlySourceConfigModule extends AbstractModule {
+  override def configure() =
+    bind(classOf[SourceConfig]).to(classOf[TestOnlySourceConfig])
+}
```

Comment:
I wonder if the difference between the two Modules might be slightly clearer if they live in the same file? Just a suggestion.